homeassistant/components/unifiprotect/const.py

```diff
@@ -75,3 +77,8 @@
 DISPATCH_ADD = "add_device"
 DISPATCH_ADOPT = "adopt_device"
 DISPATCH_CHANNELS = "new_camera_channels"
+
+EVENT_TYPE_FINGERPRINT_IDENTIFIED: Final = "fingerprint_identified"
+EVENT_TYPE_FINGERPRINT_NOT_IDENTIFIED: Final = "fingerprint_not_identified"
```

Comment:
```suggestion
EVENT_TYPE_FINGERPRINT_IDENTIFIED: Final = "identified"
EVENT_TYPE_FINGERPRINT_NOT_IDENTIFIED: Final = "not_identified"
```

I think `fingerprint_` is redundant for a fingerprint scanner